### PR TITLE
add user history to discord notification on segment reporting

### PR DESCRIPTION
**Pull request by Joe-Dowd**

### CODE REVIEW

File: src/routes/voteOnSponsorTime.js [hunk 1]
```diff
@@ -86,7 +86,11 @@ module.exports = async function voteOnSponsorTime(req, res) {
       // Send discord message
       if (type != 1) {
           // Get video ID
-          let submissionInfoRow = db.prepare("SELECT videoID, userID, startTime, endTime FROM sponsorTimes WHERE UUID = ?").get(UUID);
+          let submissionInfoRow = db.prepare("SELECT s.videoID, s.userID, s.startTime, s.endTime, u.userName, "+
+            "(select count(1) from sponsorTimes where userID = s.userID) count, "+
+            "(select count(1) from sponsorTimes where userID = s.userID and votes <= -2) disregarded "+
```
Comment (TurfJakkals) [inline src/routes/voteOnSponsorTime.js:91]: Instead of 'disregarded", maybe use "ignored" instead?

File: src/routes/voteOnSponsorTime.js [hunk 2]
```diff
@@ -104,15 +108,18 @@ module.exports = async function voteOnSponsorTime(req, res) {
...
+                                  + " Views**\n\n**Submission ID:** " + UUID 
+                                  + "\n\n**Submitted by:** "+submissionInfoRow.userName+"\n " + submissionInfoRow.userID 
+                                  + "\n\n**Total User Submissions:** "+submissionInfoRow.count
+                                  + "\n**Disregarded User Submissions:** "+submissionInfoRow.disregarded
+                                  +"\n\n**Timestamp:** " + 
                                       getFormattedTime(submissionInfoRow.startTime) + " to " + getFormattedTime(submissionInfoRow.endTime),
                               "color": 10813440,
                               "author": {
-                                  "name": userSubmissionCountRow.submissionCount === 0 ? "Report by New User" : (vipRow.userCount !== 0 ? "Report by VIP User" : "")
+                                  "name": userSubmissionCountRow.submissionCount === 0 ? "Report by New User" : (vipRow.userCount !== 0 && "Report by VIP User")
```
Comment (ajayyy) [inline src/routes/voteOnSponsorTime.js:122]: Won't this display "false"?

Comment (Joe-Dowd) [reply src/routes/voteOnSponsorTime.js:122]: you're right i forgot to move the close bracket

File: src/routes/voteOnSponsorTime.js (hunk 1 above)
Comment (Joe-Dowd) [reply src/routes/voteOnSponsorTime.js:91]: will do

File: src/routes/voteOnSponsorTime.js (hunk 2 above)
Comment (Joe-Dowd) [reply src/routes/voteOnSponsorTime.js:122]: and the fact i didn't even need to change it

File: src/routes/voteOnSponsorTime.js (hunk 1 above)
Comment (Joe-Dowd) [reply src/routes/voteOnSponsorTime.js:91]: Changed it on line 117

**Comment by ajayyy** [comment]
This breaks for people who have not set a username. It should be `left join` instead of `inner join`. I have changed this in my PR.

**Pull request merged**